pkgs/development/python-modules/djangosaml2/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, buildPythonPackage, fetchFromGitHub, defusedxml, django, pysaml2
+, pythonOlder, }:
+
+buildPythonPackage rec {
+  pname = "djangosaml2";
+  version = "1.9.3";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "IdentityPython";
+    repo = "djangosaml2";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-rbmEJuG2mgozpCFOXZUJFxv8v52IRQeaeAKfeUDACeU=";
+  };
+
+  propagatedBuildInputs = [ django defusedxml pysaml2 ];
+
+  pythonImportsCheck = [ "djangosaml2" ];
+
+  checkPhase = ''
+    runHook preCheck
+
+    python tests/run_tests.py
+
+    runHook postCheck
+  '';
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
https://github.com/NixOS/nixpkgs/issues/208242